commit 5ef4017a72f77f0bc07d8efb35c140b42e064f70
Author: Bob Copeland <me@bobcopeland.com>
Date:   Thu Jan 22 08:44:19 2009 -0500

    ath5k: remove unused led_off parameter
    
    ath5k_softc->led_off hasn't been used since commit
    3a078876caee9634dbb9b41e6269262e30e8b535, "convert LED code to use
    mac80211 triggers."
    
    Changes-licensed-under: 3-Clause-BSD
    
    Reported-by: Tobias Doerffel <tobias.doerffel@gmail.com>
    Signed-off-by: Bob Copeland <me@bobcopeland.com>
    Acked-by: Nick Kossifidis <mickflemm@gmail.com>
    Signed-off-by: John W. Linville <linville@tuxdriver.com>

diff --git a/drivers/net/wireless/ath5k/base.h b/drivers/net/wireless/ath5k/base.h
index facc60d..c0fb8b5 100644
--- a/drivers/net/wireless/ath5k/base.h
+++ b/drivers/net/wireless/ath5k/base.h
@@ -148,8 +148,7 @@ struct ath5k_softc {
 	u8			bssidmask[ETH_ALEN];
 
 	unsigned int		led_pin,	/* GPIO pin for driving LED */
-				led_on,		/* pin setting for LED on */
-				led_off;	/* off time for current blink */
+				led_on;		/* pin setting for LED on */
 
 	struct tasklet_struct	restq;		/* reset tasklet */
 
